src/App.vue

```diff
@@ -81,23 +81,26 @@
       </div>
     </transition>
 
-    <sbc-header />
+    <SbcHeader />
+
     <PaySystemAlert />
+
     <div class="app-body">
       <main v-if="!isErrorDialog">
-        <entity-info />
+        <EntityInfo />
 
-        <v-container class="view-container pt-8">
+        <v-container id="container-main" class="py-8">
```

Comment:
The class wasn't actually used but I added the id in case we need to find this element.